.gitignore

```diff
@@ -6,6 +6,13 @@ yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
 
+#node
+node_modules
+node_modules/
+
+#test.md
+TEST.md
+
 # Diagnostic reports (https://nodejs.org/api/report.html)
 report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
```

Comment:
- Kindly make sure your .gitignore is clean and contains only test.md and node modules as per the requirements :recycle: 
- Let's make sure test.md is not seen in GitHub as it is ignored by git :recycle: